### DISCUSSION TITLE
Welcome to RDA5807 Discussions!

### DISCUSSION THREAD

**Discussion by pu2clr** [pu2clr/RDA5807#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by z34pec** [comment]
Hello.
I am planning to make your FM radio using Oled display. Maybe I am searching werong but where is wiring diagram when using oled display (64x128) ???

**Comment by pu2clr** [comment]
Hello.
Please, check the folder
https://github.com/pu2clr/RDA5807/tree/master/examples/RDA5807_03_ATTINY_84_85
This folder has 10 examples that can help you to use OLED in your project.
They are aimed at ATTiny84/85. However, you can adapt them to your favorite
microcontroller.

Let me know if you have any questions about this.

73 de PU2CLR.
Ricardo.







On Sun, Jul 2, 2023 at 7:46 AM z34pec ***@***.***> wrote:

> Hello.
> I am planning to make your FM radio using Oled display. Maybe I am
> searching werong but where is wiring diagram when using oled display
> (64x128) ???
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pu2clr/RDA5807/discussions/11#discussioncomment-6336276>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAVSJNBAM7RNB6GZMCZRNPDXOFGOXANCNFSM6AAAAAAXUEWXIQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

QRZ: http://www.qrz.com/db/PU2CLR
Skype: ricardo_caratti
Mobile: +55 61 981442296

**Reply by DoruE** [reply]
Hi! I'm implementing RDA5807_02_TFT_display variant and I got a lot of errors during compiling. The first ones refer to the missing of some members of the class  RDA5807, as in tthe example bellow:
D:\Documents\Arduino\RDA5807_02_TFT_display\RDA5807_02_TFT_display.ino: In function 'void checkRDS()':
RDA5807_02_TFT_display:312:24: error: 'class RDA5807' has no member named 'getRdsProgramInformation'; did you mean 'getRdsProgramType'?
       programInfo = rx.getRdsProgramInformation();

RDA5807_02_TFT_display:313:24: error: 'class RDA5807' has no member named 'getRdsStationName'; did you mean 'getRdsVersionCode'?
       stationName = rx.getRdsStationName();

.
.
exit status 1
'class RDA5807' has no member named 'clearRdsBuffer'; did you mean 'clearRdsFifo'?

Any sugestions?

**Comment by pu2clr** [comment]
Yes.
Use the 1.1.9 version (latest version) of the Library.
I wish you success.


On Wed, Sep 4, 2024 at 8:17 AM DoruE ***@***.***> wrote:

> Hi! I'm implementing RDA5807_02_TFT_display variant and I got a lot of
> errors during compiling. The first ones refer to the missing of some
> members of the class RDA5807, as in tthe example bellow:
> D:\Documents\Arduino\RDA5807_02_TFT_display\RDA5807_02_TFT_display.ino: In
> function 'void checkRDS()':
> RDA5807_02_TFT_display:312:24: error: 'class RDA5807' has no member named
> 'getRdsProgramInformation'; did you mean 'getRdsProgramType'?
> programInfo = rx.getRdsProgramInformation();
>
> RDA5807_02_TFT_display:313:24: error: 'class RDA5807' has no member named
> 'getRdsStationName'; did you mean 'getRdsVersionCode'?
> stationName = rx.getRdsStationName();
>
> .
> .
> exit status 1
> 'class RDA5807' has no member named 'clearRdsBuffer'; did you mean
> 'clearRdsFifo'?
>
> Any sugestions?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pu2clr/RDA5807/discussions/11#discussioncomment-10544182>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAVSJNBNHYT3KZH447LQSIDZU3T4NAVCNFSM6AAAAABNUCUXP2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANJUGQYTQMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

QRZ: http://www.qrz.com/db/PU2CLR
Skype: ricardo_caratti
Mobile: +55 61 981442296
